src/main.ts

```diff
@@ -8,11 +8,10 @@ const cpExec = util.promisify(require('child_process').exec);
 
 import { createScriptFile, TEMP_DIRECTORY, NullOutstreamStringWritable, deleteFile, getCurrentTime, checkIfEnvironmentVariableIsOmitted } from './utils';
 
-const START_SCRIPT_EXECUTION_MARKER: string = `Starting script execution via docker image mcr.microsoft.com/azure-cli:`;
-const BASH_ARG: string = `bash --noprofile --norc -e `;
+const START_SCRIPT_EXECUTION_MARKER: string = "Starting script execution via docker image mcr.microsoft.com/azure-cli:";
 const AZ_CLI_VERSION_DEFAULT_VALUE = 'agentazcliversion'
 
-export async function main(){
+export async function main() {

```

Comment:
Is there any linter tool for TypeScript? A linter can help us normalize the code style and avoid distracting diffs like this. 

Just a reminder. No need to do that right now.